climada/util/test/test_coordinates.py

```diff
@@ -248,6 +248,11 @@ def test_latlon_bounds(self):
         bounds = u_coord.latlon_bounds(lat, lon, buffer=1)
         self.assertEqual(bounds, (179.5, -3.1, 186, 6))
 
+        # exceedingly large buffers are truncated to [-180, 180]
+        lat, lon = np.array([0, -2.1, 5]), np.array([-179.5, -175, -178])
+        bounds = u_coord.latlon_bounds(lat, lon, buffer=234)
+        self.assertEqual(bounds, (-180, -90, 180, 90))
```

Comment:
Before this PR, this would fail because the buffer is so large that adding it both to the right and to the left of the bounds would actually cover more than a full circle.